openedx/adg/lms/utils/date_utils.py

```diff
@@ -22,26 +22,32 @@ def month_choices(default_title=None):
     return choices
 
 
-def year_choices(from_year=MINIMUM_YEAR_OPTION, default_title=None):
+def year_choices(from_year=MINIMUM_YEAR_OPTION, default_title=None, is_upcoming=False, to_year=None):
     """
-    Create choices for all the years from `from_year` to current year, in descending order. If `from_year` is not
-    provided then range will start from 1900 i.e. [(2020, 2020) ... (1900, 1900)]
+    Create choices for previous or upcoming years depending on the is_upcoming flag.
 
     Args:
         default_title (str): The default title i.e. Year
-        from_year (int): Optional, range from current year to this year; defaults to 1900
+        is_upcoming (bool): If True, generate list of previous years; else list of upcoming years
+        to_year (int): Optional, range from current year to this year; defaults to 10 year from current year
+        from_year (int): Optional, descending range from this year to current year; defaults to 1900
```

Comment:
current year to this year and this year to current year is slightly ambiguous, instead of this year we should use to_year and from_year.